src/library/tabledelegates/keydelegate.cpp

```diff
@@ -0,0 +1,61 @@
+#include "library/tabledelegates/keydelegate.h"
+
+#include <qnamespace.h>
+
+#include <QPainter>
+#include <QStyle>
+#include <QTableView>
+
+#include "moc_keydelegate.cpp"
+#include "track/keyutils.h"
+#include "util/color/predefinedcolorpalettes.h"
+#include "util/color/rgbcolor.h"
+
+KeyDelegate::KeyDelegate(QTableView* pTableView)
+        : TableItemDelegate(pTableView) {
+}
```

Comment:
move into header please.

Reply:
What's the reason for this request?

Reply:
potentially performance. constructors this trivial will likely always get inlined, but the compiler needs it in the header for that.